src/style.css

```diff
@@ -1,3 +1,54 @@
 body {
   margin: 0;
 }
+
+.header{
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.header img{
+  margin: 10px 0;
+}
+
+.nav_list{
+  display: flex;
+  list-style: none;
+  padding: 0;
+}
+
+.nav_item {
+  display: inline-block;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 600;
+  height: 15px;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You have specified 'Roboto' as your font-family but there is no fallback font in case 'Roboto' doesn't load.